### DISCUSSION TITLE
Do we need to test against 2.3 / 2.4?

### DISCUSSION THREAD

**Discussion by ianks** [ruby/rubygems#5670] [General]
2.3 and 2.4 are very EOL at this point. In fact, I cannot even get 2.3 to install on my machine. This adds a bit of a maintenance burden, can we remove them from the CI matrix?

**Comment by deivid-rodriguez** [comment]
They will be removed by the end of this year.

**Comment by deivid-rodriguez** [comment]
Also, we should drop support at the same time we stop testing. No testing but keeping support feels wrong to me. Have a look at #3260 for more context.

**Comment by hsbt** [comment]
We only support Ruby 3.0 or higher now.
